uiowa/uiowa
# Added container query `container-type` for layout regions in uids_base

Resolves https://github.com/uiowa/uiowa/issues/6386. 
Resolves https://github.com/uiowa/uiowa/issues/4678. 

# How to test
```
ddev blt frontend && ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /cards-copy-0
```
- Code review
- Confirm timeline line is still moving on https://sandbox.uiowa.ddev.site/timeline
- Confirm dropdown menu is visible on https://sandbox.uiowa.ddev.site/about-1/example-page-horizontal-menu
- Using Safari's responsive design mode (Develop > Enter responsive design mode) scroll the entire page and observe left/right aligned cards adjust to stacked on https://sandbox.uiowa.ddev.site/cards-copy-0, look for any bugs as viewport width gets smaller.
